recipes/sample/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -x
+
+pwd
+
+./configure
```

Comment:
can you use `--prefix=${PREFIX}` and then `make install`?

Reply:
I'm not quite sure where I should put the `--prefix=${PREFIX}`
After `make`, `make install`, `./configure`?
I tried `make install` but it didn't work so moved the binary manually.

Reply:
No, that shoudl be a argument to `./configure --prefix=${PREFIX}`

E.g. https://github.com/conda-forge/libidn11-feedstock/blob/6f8c64b6c42f04bc4f5235c990538fb37599f34e/recipe/build.sh#L42-L46 etc.

Reply:
I see, thanks! Will try it. I've never done any c compilation stuff, this is all new to me.

Reply:
Seems to have worked! Thanks @wolfv!